api/src/extensions.ts

```diff
@@ -421,7 +421,10 @@ class ExtensionManager {
 		for (const hook of hooks) {
 			try {
 				const hookPath = path.resolve(hook.path, hook.entrypoint);
-				const hookInstance: HookConfig | { default: HookConfig } = await import(`file://${hookPath}?t=${Date.now()}`);
+
+				const hookInstance: HookConfig | { default: HookConfig } = await import(
+					`./${pathToRelativeUrl(hookPath, __dirname)}?t=${Date.now()}`
```

Comment:
Wasn't the `file://` prefix required? cc @br41nslug 🤔

Reply:
Ohhh or was it because the original hookPath was an absolute path?? 👀

Reply:
Yeah, `pathToRelativeUrl` creates a relative URL from the path. I changed this because, while it does seem to work in Node, having parameters on a `file://` URL does not seem to be standard compliant.